share/src/main/resources/alfresco/share-config.xml

```diff
@@ -129,7 +129,7 @@
          <!-- minimum length for username and password -->
          <username-min-length>2</username-min-length>
          <password-min-length>3</password-min-length>
-         <show-authorization-status>true</show-authorization-status>

```

Comment:
Have this false by default.

Reply:
I would say it’s OK to remove the setting - we should however call-out the change in any subsequent release notes and point to the docs or explicitly state where they can re-enable